management/src/main/java/org/kaazing/gateway/management/session/SessionManagementBeanImpl.java

```diff
@@ -202,7 +187,7 @@ public boolean areNotificationsEnabled() {
 
     @Override
     public void incrementExceptionCount() {
-        exceptionCount++;
```

Comment:
This use to be reported in JMX / SNMP if I recall correctly?  (maybe it was never implemented).  Presumably this would break that?